### PR TITLE
Config

### PR DESCRIPTION
**Pull request by tporquet**

### CODE REVIEW

**Comment by tporquet** [comment]
[ezLOGGER ++](https://trello.com/c/wX5p2yJJ/138-ezlogger)

File: src/index.html [hunk 1]
```diff
@@ -83,7 +85,8 @@
 
     <!-- Export -->
     <ui-modal :show.sync="show.export" type="normal" header="Export">
-      <p>Click below to export the traffic you captured as a standard log file. You can give it straight to ezPAARSE, <strong>no configuration needed</strong>.</p>
+      <p>Click below to export the traffic you captured as a <strong>standard log file</strong><br />
```
Comment (nojhamster) [inline src/index.html]: Why is there a line break in the middle of the sentence ?

**Comment by nojhamster** [comment]
Looks fine, apart from a little typo :)

File: src/index.html (hunk 1 above)
Comment (tporquet) [inline src/index.html]: I thought it looked better but you can get rid of it if you prefer

**Comment by nojhamster** [comment]
It seems that this PR includes #8, I guess you were on the **menu** branch when you created the **config** branch :)

**Comment by tporquet** [comment]
exactly: i'm not used to branches and pull requests yet. But i'll get better!

**Comment by nojhamster** [comment]
Well let's just merge this one and close the other ;)

**Comment by tporquet** [comment]
Thanks!

**Pull request merged**